developers/wcs/platform/billing.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Billing
+sidebar_position: 3
+# image: og/wcs/user_guides.jpg
+---
+
+Sandbox clusters are free. You do not need a billing account to create a Sandbox cluster.
+
+Serverless clusters are charged for resource usage. The monthly charge is determined by your usage and your support level. You have to have a billing account to create a serverless cluster.
+
+To estimate costs for different combinations of usage levels and SLAs, see the [pricing calculator](https://weaviate.io/services/serverless).
+
+## Create an account
+
+Weaviate Cloud (WCD) uses [Stripe](https://stripe.com/) for billing. To create a Stripe account follow these steps:
+
+1. On the Clusters tab, click the `Create cluster` button.
+1. Choose a Managed cluster.
+1. Click the `Add billing` button.<br/>
+1. Select your location on the next screen.
+1. Continue to the Stripe account creation page.<br/>
+1. Create your Stripe account.
+
+## Usage charges
+
+Usage charges reflect the number and size of the objects in your collection. Weaviate polls each collection to determine a daily object count.
+
+- If the objects are uncompressed, the billing system multiplies the object count by the number of dimensions to get the `vectorDimensionsSum`.
+- If the objects are compressed, the billing system multiplies the object count by the number of vector segments to get the `vectorSegmentsSum`.
+- Older plans may also calculate a similar value, the `queryDimensionsTotal`.
```

Comment:
Older plans should no longer be in effect for serverless clusters, so I believe this line can be omitted.

Reply:
Thanks! 

Updated